cluster-defns/1.hopsworks.yml

```diff
@@ -40,6 +40,8 @@ attrs:
     LockPagesInMainMemory: 0
     TotalMemoryConfig: 4G
     NumCPUs: 4
+  flyingduck:
+    memory_gbs: 2
     
 groups:
   namenodes:
```

Comment:
same here

Reply:
When using the flyingduck property without using the jimdowling/hopsworks-chef branch, the installer will complain:

`==> default: ERROR [2023-02-01 12:01:55,091] se.kth.karamel.webservice.KaramelServiceApplication: Inalid yaml file; Invalid attributes, all used attributes must be defined in metadata.rb files: [flyingduck/memory_gbs]`

Reply:
Will investigate.